### ISSUE TITLE
use cv::cuda::minMaxLoc in python

### ISSUE THREAD

**Issue by Dawning23** [Ikwus/qiita_post#3]
Hi, Thank you for the template matching code and the CPU/CUDA speed comparison. 

At present, I have encountered the following problems when doing template matching on GPU:

- Once the template match is complete, I need to get the position of the most appropriate point, which is the `cv.minMaxLoc` function. But I needed it to work on the GPU as well, so I tried the `cv.cuda.minMaxLoc` function like:

  ```
  e = cv2.cuda.minMaxLoc(src=template, minVal=a, maxVal=b, minLoc=c, maxLoc=d)
  ```

The code works, but it doesn't get the right results. 

The version of OpenCV I am using is 4.4.0. Could you please help me?

**Comment by Ikwus** [comment]
Thank you :)

I have also encountered same problem. 
As I shown in the jupyter notebook, I wrote and avoid `minMaxLoc` function to overcome the problem.

`gresult = matcher.match(gsrc, gtmpl)  `
`resultg = gresult.download()  `
`min_valg, max_valg, min_locg, max_locg = cv.minMaxLoc(resultg)`

I tried to use OpenCV-CUDA (python) for several purpose, but I felt that there were many things that were implemented in Cpp but not in Python, so I gave up and took a detour.
I'm sorry I can't help you.
